### ISSUE TITLE
Can't start a new track

### ISSUE THREAD

**Issue by BNAndras** [exercism/vscode-exercism-cli#8]
The documentation indicates I can pick any currently available track including a new one. However, I get an error if I download hello world for a track I'm not enrolled in. This seems to be a limitation of the Exercism CLI itself.

``` ERR [Extension Host] Error: Command failed: exercism download --exercise hello-world --track tcl
Error: You have not joined this track

    at ChildProcess.exithandler (node:child_process:408:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
console.ts:137 [Extension Host] Error: Command failed: exercism download --exercise hello-world --track tcl
Error: You have not joined this track

    at ChildProcess.exithandler (node:child_process:408:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:302:5)
y @ console.ts:137
$logExtensionHostMessage @ mainThreadConsole.ts:39
_doInvokeHandler @ rpcProtocol.ts:455
_invokeHandler @ rpcProtocol.ts:440
_receiveRequest @ rpcProtocol.ts:370
_receiveOneMessage @ rpcProtocol.ts:296
(anonymous) @ rpcProtocol.ts:161
invoke @ event.ts:622
deliver @ event.ts:824
fire @ event.ts:785
fire @ ipc.net.ts:646
_receiveMessage @ ipc.net.ts:988
(anonymous) @ ipc.net.ts:852
invoke @ event.ts:622
deliver @ event.ts:824
fire @ event.ts:785
acceptChunk @ ipc.net.ts:390
(anonymous) @ ipc.net.ts:346
E @ ipc.net.ts:70
emit @ node:events:526
addChunk @ node:internal/streams/readable:315
readableAddChunk @ node:internal/streams/readable:289
Readable.push @ node:internal/streams/readable:228
onStreamRead @ node:internal/stream_base_commons:190
```

**Comment by SleeplessByte** [comment]
Yes nice. I _think_ I can solve this by showing you the track page to join!

**Comment by SleeplessByte** [comment]
![image](https://user-images.githubusercontent.com/1964376/192771552-7e240ea2-628a-42ae-b494-bd44c482bdd9.png)

Here is the fix in `0.2.2`.
